projects/extension/tests/test_openai.py

```diff
@@ -284,6 +284,32 @@ def test_openai_chat_complete(cur, openai_api_key):
     assert actual is True
 
 
+def test_openai_chat_complete_with_new_o_models(cur, openai_api_key):
```

Comment:
BTW, tried to unify some of the tests but ended up having a bit less of lines but adding a lot of complexity penalizing readability.
Sometimes, duplication is good :)

Reply:
yeah. duplication gets a bad rap